### ISSUE TITLE
Can't open Push tab

### ISSUE THREAD

**Issue by Pikorua** [parse-community/parse-dashboard#811]
Make sure these boxes are checked before submitting your issue -- thanks for reporting issues back to Parse Dashboard!

- [X] You're running version >=1.0.23 of Parse Dashboard.

- [X] You're running version >=2.3.2 of Parse Server.

- [X] You've searched through [existing issues](https://github.com/ParsePlatform/parse-dashboard/issues?utf8=%E2%9C%93&q=). Chances are that your issue has been reported or resolved before.

#### Environment Setup
PARSE DASHBOARD 1.1.2
PARSE SERVER 2.6.5

#### Steps to reproduce
Open Parse Dashboard, click on PUSH in the left menu on the bottom.
Nothing seems to happen. 
When opening developer tools, in the console there is an error. This error happens when the push menu on the left is clicked. 

#### Logs/Trace
Note: If you get a browser JS error please run `npm run dev`. This will provide source maps and a much more useful stack trace.

dashboard.bundle.js:47 Uncaught TypeError: Cannot read property 'filter' of undefined
    at t.value (dashboard.bundle.js:47)
    at d.performInitialMount (dashboard.bundle.js:32)
    at d.mountComponent (dashboard.bundle.js:32)
    at Object.mountComponent (dashboard.bundle.js:10)
    at d.performInitialMount (dashboard.bundle.js:32)
    at d.mountComponent (dashboard.bundle.js:32)
    at Object.mountComponent (dashboard.bundle.js:10)
    at d.performInitialMount (dashboard.bundle.js:32)
    at d.mountComponent (dashboard.bundle.js:32)
    at Object.mountComponent (dashboard.bundle.js:10)
value @ dashboard.bundle.js:47
performInitialMount @ dashboard.bundle.js:32
mountComponent @ dashboard.bundle.js:32
mountComponent @ dashboard.bundle.js:10
performInitialMount @ dashboard.bundle.js:32
mountComponent @ dashboard.bundle.js:32
mountComponent @ dashboard.bundle.js:10
performInitialMount @ dashboard.bundle.js:32
mountComponent @ dashboard.bundle.js:32
mountComponent @ dashboard.bundle.js:10
updateChildren @ dashboard.bundle.js:32
_reconcilerUpdateChildren @ dashboard.bundle.js:33
_updateChildren @ dashboard.bundle.js:33
updateChildren @ dashboard.bundle.js:33
_updateDOMChildren @ dashboard.bundle.js:33
updateComponent @ dashboard.bundle.js:33
receiveComponent @ dashboard.bundle.js:33
receiveComponent @ dashboard.bundle.js:10
_updateRenderedComponent @ dashboard.bundle.js:32
_performComponentUpdate @ dashboard.bundle.js:32
updateComponent @ dashboard.bundle.js:32
receiveComponent @ dashboard.bundle.js:32
receiveComponent @ dashboard.bundle.js:10
_updateRenderedComponent @ dashboard.bundle.js:32
_performComponentUpdate @ dashboard.bundle.js:32
updateComponent @ dashboard.bundle.js:32
receiveComponent @ dashboard.bundle.js:32
receiveComponent @ dashboard.bundle.js:10
_updateRenderedComponent @ dashboard.bundle.js:32
_performComponentUpdate @ dashboard.bundle.js:32
updateComponent @ dashboard.bundle.js:32
receiveComponent @ dashboard.bundle.js:32
receiveComponent @ dashboard.bundle.js:10
_updateRenderedComponent @ dashboard.bundle.js:32
_performComponentUpdate @ dashboard.bundle.js:32
updateComponent @ dashboard.bundle.js:32
performUpdateIfNecessary @ dashboard.bundle.js:32
performUpdateIfNecessary @ dashboard.bundle.js:10
s @ dashboard.bundle.js:7
perform @ dashboard.bundle.js:12
perform @ dashboard.bundle.js:12
perform @ dashboard.bundle.js:7
S @ dashboard.bundle.js:7
closeAll @ dashboard.bundle.js:12
perform @ dashboard.bundle.js:12
batchedUpdates @ dashboard.bundle.js:33
o @ dashboard.bundle.js:7
dispatchEvent @ dashboard.bundle.js:33

**Comment by nextbigidea** [comment]
Same issue. Help!

**Comment by nextbigidea** [comment]
Update - looks like going directly to /push/activity/all and /push/audiences works, just not /push/new. Thoughts?

**Comment by nextbigidea** [comment]
Also - getting random error: 

2017-11-15T19:48:19.769Z - Invalid parameter for query: audience_limit
2017-11-15T19:48:19.765Z - Error generating response. ParseError {
  code: 102,
  message: 'Invalid parameter for query: audience_limit' }

Thoughts?

**Comment by montymxb** [comment]
@Pikorua @nextbigidea  can confirm that this is happening exclusively on **1.1.2**, **1.1.0** is unaffected. There we some recent dependency updates, something in there must have broken this :/. I'll let you know when I find out what's going on here.

**Comment by montymxb** [comment]
The issue has to do with the newly added [localized push feature](https://github.com/parse-community/parse-dashboard#configuring-localized-push-notifications). Looks like there isn't a check for when it's not present. I've opened up a PR to patch this, but in the meantime you can add an empty array of localizations to fix it.
```json
{
  "apps": [{
    "serverURL": "http://localhost:1337/parse",
    "appId": "app-id-here",
    "masterKey": "master-key-here",
    "appName": "app-name-here",
    "supportedPushLocales": []
  }],
}

```

**Comment by Pikorua** [comment]
Ah! Yes, that definitely helped! Thanks heaps @montymxb :D
Side question, should I open another "issue", it seems that this latest version keeps login me out all the time... It was not doing that before... 
Thanks again! You guys rock~

**Comment by flovilmart** [comment]
There’s a new option for the cookie based sessions, the server will log you out when restarting, is that related?

**Comment by Pikorua** [comment]
No I'm not restarting the server, i'm logging in then i would click on Push and it sends me back to the login screen or i would refresh the page and it would send me back to the login page too...

**Comment by montymxb** [comment]
Hmm, @Pikorua I would open up a separate issue so we can track that independently of this.

**Comment by Droppix** [comment]
Same issue with Parse-Dahboard v1.1.2 and Parse-server 2.6.5

Works with Parse-Dashboard v1.1.2 and Parse-server 2.2.25

**Comment by montymxb** [comment]
@Droppix if it's related to being unable to see the Push section see the answer above ^.

**Comment by Droppix** [comment]
@montymxb
Works!

Thanks

**Comment by montymxb** [comment]
Closing this out as resolved. @Pikorua if you're still having a problem with the repeated logout go ahead and open up another issue so we can follow up there.

**Comment by gateway** [comment]
Any idea when this could get pushed into a new version.. no pun intended :)

**Comment by montymxb** [comment]
Soon ideally. @flovilmart do we have anything else that would go with this
or can we just make a patch release?

**Comment by flovilmart** [comment]
Let’s check the open PR’s and see whst’s Pending, otherwise, I don’t mind

**Comment by montymxb** [comment]
@flovilmart could release after #813?

**Comment by flovilmart** [comment]
Yep, we need a review on this one.

**Comment by AneesKhanzada** [comment]
just make an empty property in your parse-dashboard-config.json
"supportedPushLocales": []

**Comment by yuvrajjsingh0** [comment]
Thank you very much! @montymxb . I was stuck on this problem for weeks. Since, I'm new in parse.

**Comment by alok-pandit** [comment]
I tried this:

{
  "apps": [{
    "serverURL": "http://localhost:1337/parse",
    "appId": "app-id-here",
    "masterKey": "master-key-here",
    "appName": "app-name-here",
    "supportedPushLocales": []
  }],
}

However, I am still facing the same error.

parse-server@2.7.2

parse-dashboard@1.1.2

I am using nvm ( nvm use --lts on ubunu 16.04).

I have added "supportedPushLocales": [] to the /home/alok/.nvm/versions/node/v8.9.4/lib/node_modules/parse-dashboard/Parse-Dashboard/parse-dashboard-config.json file.

**Comment by alok-pandit** [comment]
Never Mind. Issue resolved. I was running 'parse-dashboard --appId parsePoC --masterKey *** --serverURL "http://localhost:1337/parse" --appName parsePoC' when I should have been running just parse-dashboard.

**Comment by amitkh6** [comment]
@montymxb Can see that you have provided the solution, however I am unable to follow as to what to do where.... Pls guide, any of the member who have benefited pls guide the steps..

Pls excuse my naivety to developer world
@yuvrajjsingh0 
@Droppix
